Detectors/TPC/workflow/include/TPCWorkflow/CATrackerSpec.h

```diff
@@ -140,6 +140,5 @@ struct Config {
 /// @param tpcsectors list of sector numbers
 framework::DataProcessorSpec getCATrackerSpec(o2::tpc::reco_workflow::CompletionPolicyData* policyData, ca::Config const& specconfig, std::vector<int> const& tpcsectors);
 
-o2::framework::CompletionPolicy getCATrackerCompletionPolicy();
```

Comment:
Actually I don't understand what this does or why it was here. Is it safe to remove it?

Reply:
Good catch, I dropped it to debug something else.